client/src/pages/Search/index.js

```diff
@@ -123,6 +138,8 @@ export default function Search() {
           <SearchHeader
             updateFields={updateFields}
             searchFields={searchFields}
+            priceFilter={priceFilter}
+            setPriceFilter={setPriceFilter}
```

Comment:
prop drilling price props into the filter component
Search/index.js -> SearchHeader -> PriceFilter